components/ai_chat/resources/page/components/alerts/long_conversation_info.tsx

```diff
@@ -5,42 +5,33 @@
 
 import * as React from 'react'
 import { getLocale } from '$web-common/locale'
-import Alert from '@brave/leo/react/alert'
+import Icon from '@brave/leo/react/icon'
 import Button from '@brave/leo/react/button'
 import DataContext from '../../state/context'
 import styles from './alerts.module.scss'
 import getPageHandlerInstance from '../../api/page_handler'
 
-export default function InfoLongConversation() {
+export default function LongConversationInfo() {
   const context = React.useContext(DataContext)
 
-  const handleClearChat = () => {
+  const handleClearChat = (e: any) => {
+    e.preventDefault()
     getPageHandlerInstance().pageHandler.clearConversationHistory()
     context.dismissLongConversationInfo()
   }
 
   return (
-    <div className={styles.alert}>
-      <Alert
-        mode='full'
-        type='info'
-      >
+    <div className={styles.info}>
+      <div className={styles.infoIcon}>
+        <Icon name='info-outline' />
+      </div>
+      <div className={styles.infoText}>
         {getLocale('errorContextLimitReaching')}
-        <Button
-          slot='actions'
-          kind='plain-faint'
-          onClick={handleClearChat}
-        >
-          {getLocale('clearChatButtonLabel')}
+        {' '}
```

Comment:
what's this space for?

Reply:
<img width="105" alt="image" src="https://github.com/brave/brave-core/assets/8665427/2a7d017a-33c8-41c7-a29a-b3d226e36bc6">

Reply:
<img width="94" alt="image" src="https://github.com/brave/brave-core/assets/8665427/b3b2086b-8e58-43e6-b304-61a85c99798d">
no space

Reply:
Did you fix this another way as I don't see the space anymore

Reply:
it should be fixed another way now